### DISCUSSION TITLE
How do I fetch this query in Prisma

### DISCUSSION THREAD

**Discussion by theprantadutta** [prisma/prisma#7055] [Q&A] [answered]
Hi, Thanks for building this awesome ORM. I am having problems writing this query.

So I have a User table and a Message table. Every user can have multiple message records.

This is how my schema looks like.

```
    model User {
        id       Int       @id @default(autoincrement())
        name     String?   @db.VarChar(255)
        email    String    @unique @db.VarChar(255)
        password String    @db.VarChar(255)
        Message  Message[]
    }

     model Message {
         id       Int    @id @default(autoincrement())
         content  String @db.VarChar(255)
        authorId Int
        User     User   @relation(fields: [authorId], references: [id])
     }  
```

Now I can fetch all messages. But how do I fetch all messages with the associated user? Something like this.

``` 
   data.messages
        - content
        -users
             - name
             - email
```

Now how do I do this? What would be the query? I am using Prisma with Nextjs btw.

**Comment by danielolaviobr** [comment, marked as answer]
Hello @prantaDutta, sorry for the delayed response. Since you have a relation setup between the tables you can do something like such:

`
const response = await prisma.messages.findMany({include:{ user: true}})
`

If you need more assistance, please let me know

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
